src/ConstantSum/ConstantSum.sol

```diff
@@ -228,4 +229,23 @@ contract ConstantSum is PairStrategy {
     ) internal pure override returns (uint256[] memory) {
         return new uint256[](2);
     }
+
+    /// @inheritdoc PairStrategy
+    function _computeSwapDeltaLiquidity(
+        Pool memory,
+        bytes memory params,
+        uint256 tokenInIndex,
+        uint256,
+        uint256 amountIn,
+        uint256
+    ) internal pure override returns (uint256) {
+        if (tokenInIndex == 0) {
+            return computeSwapDeltaLiquidity(
+                amountIn, abi.decode(params, (ConstantSumParams)), true
+            );
+        }
+        return computeSwapDeltaLiquidity(
+            amountIn, abi.decode(params, (ConstantSumParams)), false
+        );
+    }
```

Comment:
This could be:

```
        return computeSwapDeltaLiquidity(
            amountIn, abi.decode(params, (ConstantSumParams)), tokenInIndex == 0
        );
```

right?

Reply:
or set as its own bool variable

Reply:
Yes, good idea.